starter-code/scripts/articleView.js

```diff
@@ -37,11 +37,14 @@ articleView.handleAuthorFilter = function() {
     //         defining. "$(this)" is using jQuery to select that element, so we can chain jQuery methods
     //         onto it.
     if ($(this).val()) {
+      $('article').hide();
+      $('article[data-author="' + $(this).val() + '"]').show();
       // TODO: If the select box was changed to an option that has a value, we need to hide all the articles,
       //       and then show just the ones that match for the author that was selected.
       //       Use an "attribute selector" to find those articles, and fade them in for the reader.
 
     } else {
+      $('article').show();
       // TODO: If the select box was changed to an option that is blank, we should
```

Comment:
this will show the articles including the template, so you'd want to add `.not('.template')`